libeos-payg/tests/manager.c

```diff
@@ -213,7 +214,7 @@ test_manager_disabled (Fixture *fixture,
   g_assert_cmpuint (0, ==, epg_provider_get_expiry_time (fixture->provider));
   g_assert_cmpuint (0, ==, epg_provider_get_rate_limit_end_time (fixture->provider));
 
-  ret = epg_provider_add_code (fixture->provider, "00000000", 0, &error);
+  ret = epg_provider_add_code (fixture->provider, "00000000", 0, &time_added, &error);
```

Comment:
All the tests which make assertions about how the expiry time changes after calling add_code should make the same assertion about the value returned in time_added.